cmd/applyconfig/applyconfig.go

```diff
@@ -86,6 +91,11 @@ func gatherOptions() *options {
 	var dryMethod string
 	dryRunMethods := strings.Join(validDryRunMethods, ",")
 	flag.StringVar(&dryMethod, "dry-run-method", string(opt.dryRun), fmt.Sprintf("Method to use when running when --confirm is not set to true (valid values: %s)", dryRunMethods))
+
+	var applyMethod string
+	applyMethods := strings.Join([]string{string(applyServer), string(applyClient)}, ",")
+	flag.StringVar(&applyMethod, "apply-method", string(opt.apply), fmt.Sprintf("Method to use when applying the config (valid values: %s). Server-side apply is always enabled for file with names start with '_SS'.", applyMethods))
```

Comment:
Why do we need it?

Reply:
In case we switch to server-side apply globally.

Reply:
discussion over slack (private channel) about my concern:
https://redhat-internal.slack.com/archives/GB7NB0CUC/p1694790437683139?thread_ts=1694777342.094659&cid=GB7NB0CUC

Reply:
I think its better to have it